pkg/agent/controller/networkpolicy/fqdn_test.go

```diff
@@ -38,6 +39,12 @@ func newMockFQDNController(t *testing.T, controller *gomock.Controller, dnsServe
 	if dnsServer != nil {
 		dnsServerAddr = *dnsServer
 	}
+	mockDnsQueryQueue := workqueue.NewTypedRateLimitingQueueWithConfig(
+		workqueue.NewTypedItemExponentialFailureRateLimiter[string](1*time.Millisecond, 100*time.Millisecond),
+		workqueue.TypedRateLimitingQueueConfig[string]{
+			Name: "fqdn",
+		},
```

Comment:
It should use a fake `Clock` to speed up enqueue, see https://github.com/antrea-io/antrea/blob/47ce51ea32ab62dbf7e6e2e7f075007823fedec3/pkg/agent/controller/ipseccertificate/ipsec_certificate_controller_test.go#L280